docs/source/conf.py

```diff
@@ -128,6 +138,15 @@
     'analytics_id': 'UA-117752657-2',
 }
 
+
+# html_context = {
```

Comment:
After the GitHub Logo issue is fixed we can use Edit on Github by simply uncommenting this.

This will lead every page to it's respective `.rst` file in docs.

Note it does not work for gallery, since `.rst` are autogenerated there.

Reply:
As a rule of thumb, we should try to avoid introducing code that we don't use, we have enough to deal with already :)
We can add this back when the Github button is fixed

Reply:
Sure, I will undo this :smile: